master/pom.xml

```diff
@@ -57,13 +57,7 @@
     <sakai.simple-xml.version>2.6.4</sakai.simple-xml.version>
     <sakai.spring.groupId>org.springframework</sakai.spring.groupId>
     <sakai.spring.artifactId>spring-core</sakai.spring.artifactId>
-    <sakai.spring.version>4.3.14.RELEASE</sakai.spring.version>
```

Comment:
These properties exist to make it easier to set specific versions without having to adjust the the actual dependencies constantly. Please leave these in place and use them accordingly.